src/contracts/Protocol/QuestionAPI.sol

```diff
@@ -84,8 +84,16 @@ contract QuestionAPI is Ownable {
     function answerQuestion(uint256 questionId, string calldata answerURL) public {
         _claimController.answer(questionId, answerURL);
     }
-    
-     function disqualifyQuestion(uint256 questionId) public onlyOwner {
+
+    /**
+     * @notice Changes the cost of creating a question
+     * @param cost The new cost of creating a question
+     */
+    function setCreateCost(uint256 cost) public onlyOwner {
```

Comment:
I don't think we need this on the QuestionAPI, I think it can just be on the _costContoller... right?  Like this function will be called very infrequently and only by admins

Reply:
Yep you're right. Had a mental error thinking we'd have to transfer ownership which is not the case. Updated!